opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -239,7 +239,7 @@ class SimulatorFullyImplicitBlackoilEbos
 
             // write the inital state at the report stage
             if (timer.initialStep()) {
-                output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
+                //output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
```

Comment:
I haven't checked. There has been other changed related to output in the flow simulator that needs to be ported to flow_ebos anyway. My suggestion is to keep it as is, and sync the output functionality when things are more settled.